### ISSUE TITLE
Login buttons disappear from admin/login when system is off

### ISSUE THREAD

**Issue by uandco** [flipboxfactory/saml-sp#186]
Hi,

Even though `enableCpLoginButtons` is set to `true` in the config (and that works when the system if ON), they disappear if the system is OFF.

Imho they should still be there as users with `Access the site when the system is off` or `Access the control panel when the system is offline` might be using SSO only.

Currently, such users can't access the control panel or website at all during maintenance as they have no way to log in.

System info:
Craft Pro 4.3.5
SAML Service Provider 4.0.4

Thanks

**Comment by dsmrt** [comment]
I agree. The reason why this is happening is due to this: https://github.com/flipboxfactory/saml-sp/issues/42

I spent some time trying to get to the bottom of this issue but had no luck. There seems to be some low level issue that is making offline mode skip over the routing that we've put in place. 

However, you can still use the IdP login action url, ie `/sso/login/request/<IdP-UID>` (which is the same as the url suggested to overwrite the `loginPath` when configuring SAML for the frontend).
